### DISCUSSION TITLE
Link broken

### DISCUSSION THREAD

**Discussion by yx3016** [rejetto/hfs#165] [Q&A]
Why did I enter the Admin panel and then log me back in after a while?

As shown below
![image](https://user-images.githubusercontent.com/81002806/226083243-3b88483a-fb6f-4375-b557-5cad9cda4588.png)

**Comment by rejetto** [comment]
once hypothesis is that you restarted HFS (or updated it), and in this case all sessions are invalidated.

**Comment by rejetto** [comment]
one hypothesis is that you restarted HFS (or updated it), and in this case
all sessions are invalidated.

**Reply by yx3016** [reply]
> one hypothesis is that you restarted HFS (or updated it), and in this case all sessions are invalidated.
> […](#)
> Message ID: ***@***.***>

The problem is that I operate normally, such as modifying configurations, adding new users, and so on. HFS was not restarted or updated.

**Comment by rejetto** [comment]
let me know if you understand WHEN it happens, because otherwise it's impossible to fix
